include/eredis.hrl

```diff
@@ -27,11 +27,18 @@
 -type parser_state() :: status_continue | bulk_continue | multibulk_continue | error_continue.
 
 %% Internal types
+-ifdef(OTP_RELEASE).
+%% OTP 21 or later
+-define eredis_queue :: queue:queue().
```

Comment:
when trying to compile:
```
==> eredis
warning: the dependency :eredis requires Elixir "~> 1.5.1" but you are running on v1.11.2
Compiling 6 files (.erl)
include/eredis.hrl:32: badly formed 'define'
include/eredis.hrl:32: badly formed 'define'
include/eredis.hrl:32: badly formed 'define'
include/eredis_sub.hrl:19: type eredis_queue() undefined
include/eredis.hrl:32: badly formed 'define'
src/eredis_client.erl:322: type eredis_queue() undefined
include/eredis.hrl:32: badly formed 'define'
```

I tried to change it to
```
-define(eredis_queue, queue:queue()).
```

And it fixed the `badly formed 'define'`, but I still get
```
==> eredis
warning: the dependency :eredis requires Elixir "~> 1.5.1" but you are running on v1.11.2
Compiling 6 files (.erl)
include/eredis_sub.hrl:19: type eredis_queue() undefined
src/eredis_client.erl:322: type eredis_queue() undefined
```

I wasn't able to fix them...

Reply:
I will fix. Sorry for posting a PR without even trying to build. :-D